### PR TITLE
Fixed out of map bounds aircraft crashing on isometric mods

### PR DESCRIPTION
**Pull request by Mailaender**
Fixes a crash reported by @GraionDilach on Discord. Followup of https://github.com/OpenRA/OpenRA/pull/17990

### CODE REVIEW

**Review by abcdefg30 — DISMISSED** [review]
Ok.

File: OpenRA.Game/Map/Map.cs [hunk 1]
```diff
@@ -853,6 +853,9 @@ public WDist DistanceAboveTerrain(WPos pos)
 			var cell = CellContaining(pos);
 			var offset = pos - CenterOfCell(cell);
 
+			if (!Contains(cell))
```
Comment (pchote) [inline OpenRA.Game/Map/Map.cs]: This should be `Ramp.Contains(cell)`. `Map.Contains` behaves differently, and is a lot slower.

Comment (Mailaender) [reply OpenRA.Game/Map/Map.cs]: Fixed.

**Review by abcdefg30 — APPROVED** [review]
Ok (attempt two).

**Review by pchote — APPROVED** [review]

**Comment by pchote** [comment]
https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/57d0d5a0892ea9f753beb01f14b3fe2afdc87d4e

**Pull request merged**